main.go

```diff
@@ -2,10 +2,10 @@ package main
 
 import (
 	"github.com/NYTimes/gizmo/server"
```

Comment:
Should this one also be updated?

Reply:
Looks like we're keeping the `NYTimes` casing for all `gizmo/server` repos, is that correct?

Reply:
I wanted to downcase all of our repos for naming consistency. 

since gizmo (and gzip and couple of others) is owned by & used by other teams, I did not touch those.